client/me/get-apps/index.jsx

```diff
@@ -1,82 +1,99 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
+import React, { PropTypes } from 'react';
+import { identity } from 'lodash';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-var MeSidebarNavigation = require( 'me/sidebar-navigation' ),
-	Main = require( 'components/main' ),
-	Button = require( 'components/button' ),
-	Card = require( 'components/card' );
+import MeSidebarNavigation from 'me/sidebar-navigation';
+import Main from 'components/main';
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
 
-module.exports = React.createClass( {
+export const GetApps = ( { translate } ) => (
+	<Main className="get-apps">
+		<MeSidebarNavigation />
+		<SectionHeader label={ translate( 'Desktop apps' ) } />
+		<Card className="get-apps__desktop">
 
-	displayName: 'GetApps',
+			<section className="get-apps__app">
+				<div className="get-apps__app-icon">
+					<img src="/calypso/images/me/get-apps-osx.svg" alt={ translate( 'Mac' ) } width="96" height="96" />
+				</div>
+				<p><strong>{ translate( 'Mac' ) }</strong><br />
+				{ translate( 'Requires 10.11 or newer' ) }</p>
+				<Button href="https://apps.wordpress.com/d/osx?ref=getapps">{ translate( 'Download', { context: 'verb' } ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'download' )` *ES Score: 19*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/8693-mobile-layout-get-apps)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.